lib/Common/Codex/Utf8Codex.cpp

```diff
@@ -360,7 +391,7 @@ namespace utf8
         }
     }
 
-    void DecodeIntoAndNullTerminate(__out_ecount(cch+1) __nullterminated char16 *buffer, LPCUTF8 ptr, size_t cch, DecodeOptions options)
+    void __cdecl DecodeIntoAndNullTerminate(__out_ecount(cch+1) __nullterminated char16 *buffer, LPCUTF8 ptr, size_t cch, DecodeOptions options)
```

Comment:
Some (maybe all) of these `__cdecl` are not needed. (I added some to my prototype because originally I attempted to use the Codex lib from external host. External host may be compiled with different default calling convention, so needed these to reconcile.) If you don't plan to make the lib available by its own, don't need these.